webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -12,7 +12,7 @@ import {
 import { notificationsDrawer as sessionStorage } from '../../../common/sessionStorage';
 import API from '../../../API';
 
-const defaultNotificationsPollingInterval = 10000;
+const defaultNotificationsPollingInterval = 999999999;
```

Comment:
can you return the value to 10000 please?
for changing the interval, you can simply use `NOTIFICATIONS_POLLING`  env variable

Reply:
Ahh! must have just slipped in on my end of day code push. Thanks